src/Compatibility/ApiDiff/Microsoft.DotNet.ApiDiff/Microsoft.DotNet.ApiDiff.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetToolMinimum)</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Verify.DiffPlex" />
```

Comment:
Our product components usually don't depend on external libraries. Can you please explain why this dependency is necessary? Is that dependency available in source-build?

Reply:
The ApiDiff tool does not need to be part of source-build.

The DiffPlex tool is heavily used in many other places in our dotnet repos. In fact, many folks from our team are often contributing to the repo of DiffPlex.

This tool is used to generate the markdown documents containing the diffs. So it's critical for the purpose of this tool.